cluster-http/src/test/scala/akka/management/cluster/MultiDcSpec.scala

```diff
@@ -59,7 +59,7 @@ class MultiDcSpec
       implicit val materializer = ActorMaterializer()
 
       val routeSettings =
-        ManagementRouteProviderSettings(selfBaseUri = s"http://126.0.0.1:$httpPortA", readOnly = false)
+        ManagementRouteProviderSettings(selfBaseUri = s"http://127.0.0.1:$httpPortA", readOnly = false)
```

Comment:
huh, how did that work?

Reply:
that is only used for links it gives out (i think)